Server/Server.Core/Server.Core.Static/Settings.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <map>
+
+namespace Server::Builder
+{
+struct ParamType
+{
+    constexpr static char const* ServerPort  = "--serverport";
+    constexpr static char const* HostAddress = "--hostaddr";
+    constexpr static char const* DBName      = "--dbname";
+    constexpr static char const* DBPort      = "--port";
+    constexpr static char const* DBUser      = "--user";
+    constexpr static char const* DBPassword  = "--password";
+};
+
+class Settings
+{
+public:
+    Settings() = default;
+
+    inline Settings& SetValue(std::pair<std::string, std::string> value)
```

Comment:
Why are you use keyword inline? 
It is best used with templates, or more specifically when specifying that it is a full specialty. 

Also if you want to make an exception to the ODR, but it's a class method, so I don't see the need.

Reply:
@memorisecodead:
This keyword has been moved here from the old setValue method in ServerBuilder. At that moment, I thought it was strange for a class method. But I decided to leave it as is.
If someone else instead of me paid attention to this keyword, I delete it.

Reply:
This would be better, because if necessary the compiler itself will make this method inline